### PR TITLE
🌐 Add Korean translation for Tutorial - Intro

### PR DESCRIPTION
**Pull request by hard-coders**
This PR translates the `ko/docs/tutorial/index.md`.

related: #2017

### CODE REVIEW

File: docs/ko/docs/tutorial/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+편집기에서 이렇게 사용하면, 모든 타입 검사, 자동완성 등 작성해야 하는 코드가 얼마나 적은지 보면서 FastAPI의 장점을 실제로 확인할 수 있습니다.
+
+---
+
+## FastAPI 설치
+
+첫 번째 단계는 FastAPI 설치입니다.
+
+자습시에는 모든 선택적 종속성 및 기능을 사용하여 설치할 수 있습니다:
```
Comment (PandaHun) [inline docs/ko/docs/tutorial/index.md:41]:
```suggestion
자습시에는 모든 선택적인 의존성 및 기능을 사용하여 설치할 수 있습니다:
```

File: docs/ko/docs/tutorial/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    pip install fastapi
+    ```
+
+    추가로 서버 역할을 하는 `uvicorn`을 설치합니다:
+
+    ```
+    pip install uvicorn
+    ```
+
+    사용하려는 각 선택적 종속성에 대해서도 동일합니다.
```
Comment (PandaHun) [inline docs/ko/docs/tutorial/index.md:70]:
```suggestion
    사용하려는 각 선택적인 의존성에 대해서도 동일합니다.
```

**Review by PandaHun — COMMENTED** [review]
It's almost there!
Consider my suggestion what i wrote.

**Review by Lily011203 — APPROVED** [review]

**Review by PandaHun — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome! Thanks for the contribution @hard-coders ! :bow: :coffee: 

And thanks for the reviews @PandaHun and @Lily011203 ! :rocket:

**Pull request merged**